x/upgrade/abci.go

```diff
@@ -12,6 +12,8 @@ import (
 	"github.com/cosmos/cosmos-sdk/x/upgrade/types"
 )
 
+var DowngradeVerified = false
```

Comment:
```suggestion
// downgradeVerified is a local variable that tells if we've already sanity checked that this binary version isn't being used against an old state.
// If so, we dont run that check every beginblock.
var downgradeVerified = false
```

I think we shouldn't export this variable.

Reply:
We need this variable in [unit-tests](https://github.com/cosmos/cosmos-sdk/pull/10407/files#diff-b62fb4ada9c3b7c1876c6ff34254c784ac935f135175b1ac6bb21b101a36b11eR458).

Reply:
This must be a private variable. If you need it outside, then either move the tests to "internal" tests (in the same package rather than `_test` package, or make a function which will return a copy.